azkaban-common/src/main/java/azkaban/executor/KeyStoreManager.java

```diff
@@ -25,13 +25,13 @@
 
 /**
  * KeyStoreManger is responsible to manage the KeyStore used during containerized execution of a
- * flow. The HadoopSecurityManager must provide the KeyStore to CredentialProvider
+ * flow. The HadoopSecurityManager must provide the KeyStore to CredentialProvider.
  */
 public class KeyStoreManager {
-  private static volatile KeyStoreManager ksmInstance = null;
+  private static volatile KeyStoreManager ksmInstance;
```

Comment:
Refer to https://www.journaldev.com/1377/java-singleton-design-pattern-best-practices-examples#static-block-initialization and see if you can use Bill Pugh Singleton Implementation instead of synchronized one.